README.md

```diff
@@ -1,2 +1,254 @@
+                    class="de.atb.context.monitoring.config.models.datasources.MessageBrokerDataSource"/>
+    </datasources>
+
+    <interpreters>
+        <interpreter id="interpreter-git">
+            <configuration type="*"
+                           parser="de.atb.context.monitoring.parser.GitParser"
+                           analyser="de.atb.context.monitoring.analyser.GitAnalyser"/>
+        </interpreter>
+    </interpreters>
+
+    <monitors>
+        <monitor id="monitor-git" datasource="datasource-git" interpreter="interpreter-git" index="index-git"/>
+    </monitors>
+</config>
+```
+
+The "monitoring-config.xml" configuration file can also be changed by using the Admin UI.
+
+
+### Description
+
+#### indexes
+
+Each index entry has the following mandatory attributes
+
+- id: The unique name of the index
+- location: The URI of the location the index is stored
+
+#### datasources
+
+Each datasource entry has the following mandatory attributes
+
+- id:The unique name of the datasource
+- type:The type of the datasource. Possible values are: file, webservice, database
+- monitor:The class of the monitor to be used. Possible values are:
+    - package de.atb.context.monitoring.monitors.database.DatabaseMonitor
+    - package de.atb.context.monitoring.monitors.file.FileSystemMonitor
+    - package de.atb.context.monitoring.monitors.file.FilePairSystemMonitor
+    - package de.atb.context.monitoring.monitors.file.FileTripletSystemMonitor
+    - package de.atb.context.monitoring.monitors.webservice.WebServiceMonitor
+- options: Options for the datasource can be entered using this value. The options are dependant on the datasource to be used
+- uri:The uri of the data source to be monitored
+- class:The following datasource implementations are available
+    - package de.atb.context.monitoring.config.models.datasources.DatabaseDataSource
+    - package de.atb.context.monitoring.config.models.datasources.FilePairSystemDataSource
+    - package de.atb.context.monitoring.config.models.datasources.FileSystemDataSource
+    - package de.atb.context.monitoring.config.models.datasources.FileTripletSystemDataSource
+    - package de.atb.context.monitoring.config.models.datasources.WebServiceDataSource
+
+#### interpreters
+
+Each interpreter entry has the following mandatory attributes
+
+- id: The unique name of the interpreter
+- configuration
+    - analyser: The analyser class to be used. The following implementations are available:
+        - package de.atb.context.monitoring.analyser.database.DatabaseAnalyser
+        - package de.atb.context.monitoring.analyser.file.FileAnalyser
+        - package de.atb.context.monitoring.analyser.file.FilePairAnalyser
+        - package de.atb.context.monitoring.analyser.file.FileTripletAnalyser
+        - package de.atb.context.monitoring.analyser.webservice.WebServiceAnalyser
```

Comment:
The analysers / parsers added by SmartCLIDE should be listed as well.

Reply:
[x] done